modules/serverless-creating-a-kafka-event-sink.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-event-sinks.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-creating-a-Kafka-event-sink_{context}"]
+
+= Creating a Kafka event sink
+
+As a developer, you can create an event sink to receive events from a particular source and send them to a kafka topic. 
+
+.Prerequisites
+* You have installed the Red Hat OpenShift Serverless operator, with Knative Serving, Knative Eventing, and Knative Kafka APIs, from the Operator Hub.
+
+* You have created a Kafka topic in your Kafka environment.
+
+.Procedure
+. In the developer perspective, navigate to the *+Add* view.
+. Click *Event Sink* in the *Eventing catalog*.
+. Search for KafkaSink in the catalog items and click it.
```

Comment:
Maybe add ticks here for `KafkaSink`? 

. Search for \`KafkaSink\` in the catalog items and click it.

Reply:
Anything that you see in the GUI should be in Bold. If a user needs to fill up a term, say in a form or something, it should be in back ticks. So if KafkaSink is an option in a drop-down, maybe reword it to - Search for *KafkaSink* in the catalog items list...
If the user needs to type it to search for it then as Gabriel said, use backticks.